Login_SpotifyUI/Login_SpotifyUI/Controller/HomeViewController.swift

```diff
@@ -84,12 +85,15 @@ class HomeViewController: UIViewController {
     }
     
     @objc func addPlayListButtonClicked() {
+        guard let realm = self.realm else { return }
         guard let register = self.register else { return }
         let randomCount = Int.random(in: 1...1000)
-        let newMusic = Music(title: "\(randomCount)", artist: "test-artist")
+        let newMusic = Music(identification: register.identification, title: "\(randomCount)", artist: "test-artist")
         let realmData = RealmData(realm: realm)
+        
         realmData.addPlayList(identifier: register.identification, newMusic: newMusic)
-        filteredData = Array(register.playList)
+        filteredData = register.getPlayListArray()
```

Comment:
```suggestion
        filteredData = register.getPlayList()
```

(참고)
@DONOTINTO 

- UI에 사용되는 객체(UIImage, UITextField...)를 제외하고는 함수 혹은 변수명에 가능하면 타입 이름 자체(Int, Array, String...)을 명시하는 것은 지양하는 것이 좋습니다.
    - [Swift 공식 네이밍 컨벤션 참고](https://www.swift.org/documentation/api-design-guidelines/#naming)
- 우리는 play"List" 에서 "List" 라는 단어를 통해 `목록` 즉, `배열(Array)`임을 이미 명시해주었습니다.
- 거기에 `Array`를 뒤에 붙인다면 배열임을 나타내는 단어를 중복해서 사용하게 된 것입니다.
   - 의미가 중복되는 단어를 여러 번 사용하여 불필요하게 네이밍을 길게 늘릴 필요 없습니다.

- 알고리즘 문제에는 특정 변수명에 의미를 부여할 필요가 없기 떄문에 네이밍에 `testArray`, `testString` 과 같이 작성하곤 합니다.